patches/@nozbe+watermelondb+0.25.5.patch

```diff
@@ -27,27 +38,37 @@ index 96114ec..ecfe3c1 100644
  
    prepareDestroyPermanently(): this
 diff --git a/node_modules/@nozbe/watermelondb/Model/index.js b/node_modules/@nozbe/watermelondb/Model/index.js
-index b0e3a83..57ecb8d 100644
+index b0e3a83..1bbce74 100644
 --- a/node_modules/@nozbe/watermelondb/Model/index.js
 +++ b/node_modules/@nozbe/watermelondb/Model/index.js
-@@ -81,7 +81,7 @@ var Model = /*#__PURE__*/function () {
+@@ -81,7 +81,17 @@ var Model = /*#__PURE__*/function () {
    _proto.prepareUpdate = function prepareUpdate(recordUpdater = _noop.default) {
      var _this = this;
  
 -    (0, _invariant.default)(!this._preparedState, "Cannot update a record with pending changes");
-+    (0, _invariant.default)(!this._preparedState, "Cannot update a record with pending changes in table " + _this.table);
++    if ('deleted' === this._raw._status) {
++      console.debug("Updating a deleted record in table " + _this.table)
++      return this;
++    }
++
++    //(0, _invariant.default)(!this._preparedState, "Cannot update a record with pending changes");
++    if (this._preparedState) {
++      console.debug("Updating a record with pending changes in table " + _this.table)
++    } else {
++      this.__original = Object.assign({}, this._raw);
```

Comment:
I guess this updates `original` to what was modified last, right ?

Reply:
Original stays as the "most original value", as in before any change. I guessed it was the safest thing to do.